src/site/filters/tests/phoneNumberArrayToObject.spec.js

```diff
@@ -3,6 +3,36 @@ import { expect } from 'chai';
 const phoneNumberArrayToObject = require('../phoneNumberArrayToObject');
 
 describe('phoneNumberArrayToObject', () => {
+  it('returns empty object if data is null', () => {
+    const badData = null;
+    const expected = {};
+    expect(phoneNumberArrayToObject(badData)).to.eql(expected);
+  });
+
+  it('returns empty object if data is empty', () => {
+    const badData = [];
+    const expected = {};
+    expect(phoneNumberArrayToObject(badData)).to.eql(expected);
```

Comment:
one suggestion, `to.be.empty` instead

Reply:
In this case I want it to be an empty object specifically.